README.md

```diff
@@ -100,9 +100,10 @@ Main `ValidationError` class, extending native JavaScript `Error`.
 #### Arguments
 
 - `message` - _string_; error message (required)
-- `details` - _Array<zod.ZodIssue>_; error details (optional)
+- `options` - _ErrorOptions_; error options - see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#options) for further info (optional)
+  - `options.cause` - _ZodError_; original zod error or any other error (optional)
```

Comment:
[question] Since this accepts `ZodError` or any other error should we

```suggestion
  - `options.cause` - _Error_; original zod error or any other error (optional)
```

Reply:
It's a valid point. According to the spec, `cause` can be anything. It doesn't need to be an error. So maybe leave it as `any`?

Reply:
That could work as well 👍